content/guides/printing/pprint.adoc

```diff
@@ -0,0 +1,161 @@
+= clojure.pprint
+Sameer Rahmani
+2016-09-07
+:type: guides
+:icons: font
+:status: published
+
+== Basics
+Pretty printing is primarily implemented with the function pprint. pprint takes a single argument and formats it according to the settings of several special variables.
+Generally, the defaults are fine for pretty printing and you can simply use:
+
+[source,clojure]
+----
+(pprint obj)
+----
+
+to print your object. If you wish to write to another stream besides *out*, you can use:
+`(write obj :pretty true :stream foo)` where foo is the stream to which you wish to write.
+(The write function has a lot more options which are not yet documented. Stay tuned.)
+
+When at the REPL, the pp macro pretty prints the last output value. This is useful when you get something too complex to read comfortably. Just type:
+
+[source,clojure]
+----
+user=> (pp)
+----
+
+and you'll get a pretty printed version of the last thing output (the magic variable *1). Dispatch tables and code formatting The behavior of the pretty
```

Comment:
code font for *1, should also probably link to https://clojure.org/reference/repl_and_main or the repl guide

Reply:
Also the "Dispatch tables and code formatting" here looks like a buried header?